src/schema/submission.yaml

```diff
@@ -1301,21 +1306,21 @@ slots:
 
   geographical_region_term:
     description: >-
-      The geographical region where the Individual is located.
+      The geographical region where the Individual is located. The Geographical Region is captured using a concept from the NCIT "country" class (NCIT:C25464) (e.g., Austria, Germany, Italy).
```

Comment:
We may again shorten. We do not need to repeat "The Geographical Region" in the second sentence.